### DISCUSSION TITLE
Does Gephi support hsv color wheel?

### DISCUSSION THREAD

**Discussion by juliannej** [gephi/gephi#2370] [Q&A]
I want to alter both color and transparency of nodes. Does Gephi have any inbuilt support for the hsv color wheel?

Thanks,
Julianne

**Comment by juliannej** [comment]
Specifically for generating node color palettes? I can't seem to find a way to alter node opacity. Do you have any idea how to do this?

**Comment by eduramiba** [comment]
Gephi uses the Java-native color chooser, so sometimes the UI depends on the OS, what is your OS?

**Comment by juliannej** [comment]
Mac OS 10.13

**Comment by juliannej** [comment]
I don't see how I would do this because all the node color appearance option allows me to do is generate palettes. is there another place from which to control color of nodes?

**Comment by eduramiba** [comment]
I see, not sure, I don't own a Mac, but this is what I get doing right click on the palette color in linux:

![imagen](https://user-images.githubusercontent.com/197285/43896869-98143be4-9bda-11e8-993f-cf37d67870f6.png)

**Comment by eduramiba** [comment]
Once the palette is generated, right click the color small square in the color list.

**Comment by juliannej** [comment]
Okay. My color picker isn't so advanced when I right-click. But thank you for showing me this functionality. Is there a way to alter node opacity instead?

**Comment by eduramiba** [comment]
I think you can edit the alpha component of node colors in the edit window (with the edit tool) and also right click menu in data laboratory.
